===================================================================
RCS file: /afs/andrew.cmu.edu/system/cvs/src/cyrus/imap/backend.c,v
retrieving revision 1.39
retrieving revision 1.40
diff -u -r1.39 -r1.40
--- imap/backend.c	2006/02/07 20:57:27	1.39
+++ imap/backend.c	2006/04/03 16:34:36	1.40
@@ -39,7 +39,7 @@
  * OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-/* $Id: backend.c,v 1.39 2006/02/07 20:57:27 murch Exp $ */
+/* $Id: backend.c,v 1.40 2006/04/03 16:34:36 murch Exp $ */
 
 #include <config.h>
 
@@ -271,7 +271,7 @@
     /* need to (re)establish connection to server or create one */
     int sock = -1;
     int r;
-    int err = 0;
+    int err = -1;
     struct addrinfo hints, *res0 = NULL, *res1 = NULL, *res;
     struct sockaddr_un sunsock;
     char buf[2048], *mechlist = NULL;
